docs/topics/snippets.md

```diff
@@ -32,15 +32,15 @@ class Advert(models.Model):
         return self.text
 ```
 
-The `Advert` model uses the basic Django model class and defines two properties: text and URL. The editing interface is very close to that provided for `Page`-derived models, with fields assigned in the `panels` property. Snippets do not use multiple tabs of fields, nor do they provide the "save as draft" or "submit for moderation" features.
+The `Advert` model uses the basic Django model class and defines two properties: text and URL. The editing interface is very close to that provided for `Page`-derived models, with fields assigned in the `panels` (or `edit_handler`) property. Snippets do not use multiple tabs of fields, nor do they provide the "save as draft" or "submit for moderation" features.
```

Comment:
The last line should've been updated to reflect the current state. I could do that now or as part of the docs update sometime during the feature freeze

Reply:
Oh never mind, I already did in https://github.com/wagtail/wagtail/commit/6cb13f9c6f727e1042da2e15c59d1f0ad93f401c